RocketChat/Rocket.Chat
# [FIX] Omnichannel Business Hours form is not being rendered

* When the managers access the Business Hours Page and the configuration type = Single, just an error message is displayed.
* When the business hour configuration type = multiple, managers just can't access the business hour form, just a blank page shows up.





## Proposed changes (including videos or screenshots)
I did a small refactor on components to work better with routes and to check new changes on all fields of the form, I also put some missing translations.

### Single

#### Before
![image](https://user-images.githubusercontent.com/2493803/103356825-8611ed80-4a90-11eb-9a92-5702c8702bbb.png)

Missing translations
![image](https://user-images.githubusercontent.com/2493803/103356864-9c1fae00-4a90-11eb-92c9-4426dd5a0fde.png)

#### After
![image](https://user-images.githubusercontent.com/2493803/103356672-261b4700-4a90-11eb-816e-52504aded603.png)


### Multiple

#### Before
![image](https://user-images.githubusercontent.com/2493803/103357467-0553f100-4a92-11eb-9cfb-613d5d8d9109.png)
![image](https://user-images.githubusercontent.com/2493803/103357546-3502f900-4a92-11eb-96fc-9547f53fd679.png)

On edit, some fields are not recognizing changes and save button isn't be able to click.
![Rocket Chat](https://user-images.githubusercontent.com/2493803/103357777-c1adb700-4a92-11eb-9064-7853636ae52f.gif)


#### After
![image](https://user-images.githubusercontent.com/2493803/103356531-dc326100-4a8f-11eb-867f-e85bf935d79d.png)
![image](https://user-images.githubusercontent.com/2493803/103356578-f5d3a880-4a8f-11eb-8b8a-dc1af944b2c8.png)

Recognizing changes in all fields.
![Rocket Chat (2)](https://user-images.githubusercontent.com/2493803/103358258-eeae9980-4a93-11eb-95e7-b5560f8ad381.gif)






## Issue(s)


## Steps to test or reproduce

Single configuration: Administration panel > Omnichannel > Business Hour > Change to Single
After this, Omnichannel panel > Click twice on Business Hour link or try to save, you'll see an error.

Multiple configuration:  Administration panel > Omnichannel > Business Hour > Change to Multiple
After this, Omnichannel panel > Business Hour > 
Try to create a new one, you'll an empty page.
Try to create edit an existing, you'll that validation isn't working well.

## Further comments
